pkgs/games/terraria-server/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ file ];
-  nativeBuildInputs = [ autoPatchelfHook unzip ];
+  nativeBuildInputs = [ autoPatchelfHook stdenv.cc.cc.libgcc unzip ];
```

Comment:
This is a runtime dependency, so it should go in `buildInputs`.

Reply:
Also it seems like we don't need `file`, only `zlib`.

Reply:
right.. naming (and understanding naming) is hard..